src/plus/webviews/graph/graphWebview.ts

```diff
@@ -173,6 +173,7 @@ const defaultGraphColumnsSettings: GraphColumnsSettings = {
 	author: { width: 130, isHidden: false },
 	datetime: { width: 130, isHidden: false },
 	sha: { width: 130, isHidden: false },
+	changes: { width: 130, isHidden: false },
```

Comment:
Because of the perf impact, lets hide this by default (at least for now)

Reply:
Done in commit: [Hides the column by default](https://github.com/gitkraken/vscode-gitlens/pull/2494/commits/53c3f6009a2f81dc6642a149e985de9751d4128c)